### PR TITLE
fix(components): [el-checkbox] remove the height in normal mode

### PR DESCRIPTION
**Pull request by msidolphin**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/examples/checkbox/basic.vue [hunk 1]
```diff
@@ -1,39 +1,9 @@
...
-      size="small"
-      disabled
-    ></el-checkbox>
-    <el-checkbox
-      v-model="checked6"
-      label="Option 2"
-      size="small"
-      disabled
-    ></el-checkbox>
-  </div>
```
Comment (JeremyWuuuuu) [inline docs/examples/checkbox/basic.vue:27]: Why are we deleting the cases here?

File: packages/components/table/src/table-column/render-helper.ts [hunk 2]
```diff
@@ -118,8 +118,11 @@ function useRender<T>(
           children = originRenderCell(data)
         }
         const prefix = treeCellPrefix(data)
+        const classList = ['cell'].concat(
+          column.type === 'selection' ? ['is-selection'] : []
+        )
         const props = {
-          class: 'cell',
+          class: classList.join(' '),
```
Comment (JeremyWuuuuu) [inline packages/components/table/src/table-column/render-helper.ts:125]: **Suggestion:**  Why not:

```ts
const kls = {
  cell: true,
  'is-selection': column.type === 'selection',
  'el-tooltip': column.showOverflowTooltip,
}
```

**Comment by msidolphin** [comment]
@JeremyWuuuuu done

**Comment by JeremyWuuuuu** [comment]
@msidolphin You have conflict code please resolve them. Thanks

**Comment by msidolphin** [comment]
> @msidolphin You have conflict code please resolve them. Thanks

This proposal has been deprecated

**Pull request closed without merging**